app/utils/whitelabel.ts

```diff
@@ -98,6 +99,7 @@ services and re-entry programs.`,
   showHeaderQrCode: false,
   showMobileNav: true,
   showSearch: true,
+  showReportCrisis: true,
```

Comment:
Yep, I think that is definitely the case

Reply:
Thanks, I wasn't sure which versions of the site these defaults were servicing

Reply:
Oops my bad for not elaborating. While we do want to set this as false for the `whiteLabelDefaults` -- which is sort of a base object designed to avoid repetition in our different white label objects -- we want to overwrite that value in the `SFServiceGuide` and in the `defaultWhiteLabel` const. The `SFServiceGuide` whitelabel maps to our prod site www.sfserviceguide.org and the `defaultWhiteLabel` is our dev environment. As this stands now, the button won't show up on our prod site.

Sorry this is a bit confusing, maybe we can make this code somehow more descriptive in a future ticket.

Reply:
Ahh thanks, I think I got it right this time! Yeah, I think I was confused mostly by the multiple defaults. There may be a better was to control it by environment.